tests/Fixer/Phpdoc/NoSuperfluousPhpdocTagsFixerTest.php

```diff
@@ -2528,6 +2528,30 @@ static function ($foo): int { return 1; };',
 /** @return int */
 static fn ($foo): int => 1;',
         ];
+
+        yield 'multiline @param must be kept even if there is no description on the phpdoc tag line' => [
+            <<<'EOD'
```

Comment:
Not a big deal but other test cases here don't use NOWDOC syntax. I would not use it for new cases for consistency.

(Maybe we should create a new fixer for that.)

Reply:
@julienfalque #7665 😉

Reply:
yes :)